src/design-system/layout/footer/footer.module.scss

```diff
@@ -1,3 +1,54 @@
 .footer {
   display: flex;
+  flex-direction: column;
+  height: 200px;
+  font-family: "Poppins", sans-serif;
+}
+
+.navLinks {
+  display: flex;
+  list-style-type: none;
+  width: 90%;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 60px 0 20px;
+  border-bottom: 2px solid white;
```

Comment:
wszystkie kolorki używajmy ze zmiennych, jak nie ma to dodaj do `src/design-system/styles/global.scss`

czyli zamiast `white` daj `var(--colorWhite)`